pages/_app.tsx

```diff
@@ -21,6 +22,10 @@ class OdysseyApp extends App {
     if (process.env.NODE_ENV === "production") {
       hotjar.initialize("1547187");
     }
+
+    Sentry.init({
+      dsn: "https://1a18ac7b9aa94cb5b2a8c9fc2f7e4fc8@sentry.io/1801129"
+    });
```

Comment:
One thing; I'm not sure what happens if this function gets called twice in the same context. We should run this somewhere where its guaranteed to run on the client